src/open_inwoner/cms/utils/page_display.py

```diff
@@ -0,0 +1,45 @@
+"""Utilities for determining whether CMS pages are published"""
+
+
+from cms.models import Page
+
+from open_inwoner.cms.cases.cms_apps import CasesApphook
+from open_inwoner.cms.collaborate.cms_apps import CollaborateApphook
+from open_inwoner.cms.inbox.cms_apps import InboxApphook
+
+cms_apps = {
```

Comment:
Since you did this for some other apphooks apart from inbox (which has to do with the specific PR), maybe you should add the remaining apphooks as well (products, profile etc.).

And I am not sure if we do need separate-additional functions for each apphook. Maybe a bit different name of the main function `_is_published` could be a solution in order not to confuse this with cms's Page model method.

Reply:
I'll update the module for the other apphooks.

The idea was to use a single private function for the implementation and several public functions for the interface. `_is_published` should never be used outside these public functions. I deliberately used a name that's close to the naming used in the CMS Page model. Do you still see the danger of confusion here? In that case I can rename it.

Reply:
- Yes you are right, no need for adding the rest of the apphooks for now.

- There is no need to rename the function, this would be good to do it if we used only one function. Basically, I just wanted to understand why we created a function for each apphook separately instead of just passing the str to a single function. But you have explained this to me so you can keep it the way you have it.